### ISSUE TITLE
Deleting sketches does not work

### ISSUE THREAD

**Issue by ssmid** [SimonBuxx/LogiJS#82]
Since I'm already here: I could not delete any sketches in my account. After I click on "Delete" and "SURE?" in the dashboard, it reloads, but all sketches are still there.

Btw, I had a lot of fun today, thanks for that :)

**Comment by SimonBuxx** [comment]
You're absolutely right, this doesn't seem to work in Firefox for some reason. I'm developing in Chrome so sometimes some browser-specific bugs happen. I'll have a look at this.
Glad you had fun, I rarely get feedback so i appreciate that very much!

Simon
